pyproject.toml

```diff
@@ -49,6 +49,7 @@ dev = [
   "mypy==0.991",
   "numpy==1.22.2",
   "pre-commit>=3.2.0",
+  "psycopg2>=2.8.4",
```

Comment:
given it's an open source library, should we make the postgres stuff optional? and then also make the CI run against both postgres and sqlite?

Reply:
This is dependency *is* optional. I don't think there's any real value in running tests against both PG and sqlite. We don't have *that* much in xocto that depends on the database, but these fields absolutely depend on postgres. It'll be noted in the docs (once written).

Is there value in namespacing the `fields` folder into a `fields/postgres` structure so folks don't unintentionally import something they can't use?

Reply:
ah it is too, sorry, didn't see which list it was in.

The namespacing sounds like a reasonable idea

> don't think there's any real value in running tests against both PG and sqlite.
mmmkay